### PR TITLE
feat: UILD-333: STORY: Duplicate Instance

### PR DESCRIPTION
**Pull request by s3fs**
https://folio-org.atlassian.net/browse/UILD-333

### CODE REVIEW

File: src/common/helpers/recordFormatting.helper.ts [hunk 1]
```diff
@@ -209,3 +209,15 @@ export const getNonBFMapElemByContainerKey = (
 
   return { key: groupElemKey, value: groupElemValue };
 };
+
+export const applyIntlToTemplates = ({
+  templates,
+  fmt,
```
Comment (SKarolFolio) [inline src/common/helpers/recordFormatting.helper.ts:215]: `fmt`? What is it? Please, give variables readable names.

File: src/common/helpers/schema.helper.ts [hunk 2]
```diff
@@ -6,6 +6,54 @@ import {
...
+  reference?: RecordReference;
+  recordBlocks: RecordBlocksList;
+  selectedEntriesService: ISelectedEntries;
+  schemaWithDuplicatesService: ISchemaWithDuplicates;
+  userValuesService: IUserValues;
+  commonStatusService: ICommonStatus;
+  template?: ResourceTemplateMetadata[];
+};
+
+export const getSchemaAndUserValuesFromRecord = async ({
```
Comment (SKarolFolio) [inline src/common/helpers/schema.helper.ts:25]: Not sure if the helper is the best place to do this.
We have `ServicesContext` and custom hooks for this purpose.
Please add a comment that this code should be refactored.

**Review by SKarolFolio — APPROVED** [review]
Please take a look at the comments.

File: src/common/helpers/recordFormatting.helper.ts (hunk 1 above)
Comment (s3fs) [reply src/common/helpers/recordFormatting.helper.ts:215]: `fmt` is a common shorthand for `format`

**Review by askhat-abishev — APPROVED** [review]

**Pull request merged**